src/libraries/IterableMap.sol

```diff
@@ -0,0 +1,104 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+
+contract IterableMap {
```

Comment:
We should probably name this contract (and file) `IterableMapUint256` or something.  I suspect we will want/need other iterable maps in the future, and having them named such that we can easily spin them up with some simple copy/paste would be nice.